include/aspect/material_model/reactive_fluid_transport.h

```diff
@@ -179,6 +179,10 @@ namespace aspect
         std::vector<double> csat_sediment_poly_coeffs {-0.150662, 0.301807, 1.01867};
         std::vector<double> Td_sediment_poly_coeffs {2.83277, -24.7593, 85.9090, 524.898};
 
+        // The polynomials breakdown above certain pressures, 10 GPa for peridotite, 26 GPa for gabbro, 16 GPa for MORB,
+        // and 50 GPa for sediment.
+        std::vector<double> pressure_cutoffs {10, 26, 16, 50};
```

Comment:
can you make this const?

Reply:
Are these cutoff pressures reported in the Tian et al. paper? If yes, can you add a sentence to the documentation above to indicate this?

Reply:
As far as I could see these values are not reported in the Tian et al., paper, but these values could definitely be made constant yes